service/create/service.go

```diff
@@ -1349,17 +1319,31 @@ func (s *Service) updateFunc(oldObj, newObj interface{}) {
 		return
 	}
 
-	asg := awsresources.AutoScalingGroup{
-		Client:  clients.AutoScaling,
-		Name:    fmt.Sprintf("%s-%s", key.ClusterID(cluster), prefixWorker),
-		MinSize: newSize,
-		MaxSize: newSize,
+	// Process the Auto Scaling Group for the workers.
```

Comment:
Here the stack is updated. For now, just the updatable parameters are provided.

This should be replaced by calling the processCluster method that the addFunc calls. However there is a bug due to the high volume of update events. The runInstances method that launches the master isn't threadsafe and 2 masters are created.